src/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -242,23 +245,42 @@ public RemoteOperationResult(boolean success, HttpMethod httpMethod) throws IOEx
                 }
             }
         }
+
+        if (mHttpCode == HttpStatus.SC_FORBIDDEN) {
+            String bodyResponse = httpMethod.getResponseBodyAsString();
+
+            if (bodyResponse != null && bodyResponse.length() > 0) {
+                InputStream is = new ByteArrayInputStream(bodyResponse.getBytes());
+                ErrorMessageParser xmlParser = new ErrorMessageParser();
+                try {
+                    String errorMessage = xmlParser.parseXMLResponse(is);
+                    if (errorMessage != "" && errorMessage != null) {
```

Comment:
You can use isEmpty() method to check it, but if you change it, be sure to change the order:
if(errorMessage != null && !errorMessage.isEmpty())